build_scripts/ubuntu_sim_nuttx.sh

```diff
@@ -45,9 +46,10 @@ else
     sudo dpkg --add-architecture i386
     sudo apt-get update
     sudo apt-get install libc6:i386 libgcc1:i386 libstdc++5:i386 libstdc++6:i386 -y
-    sudo apt-get install gcc-5.4-base:i386 -y
+    sudo apt-get install gcc-7.3-base:i386 -y
```

Comment:
@dagar This is an incorrect guess. What do I use for the 32 bit libraries?

Reply:
I don't believe this package exists, however we don't have to have the regular gcc version with the arm-none-eabi-gcc version.

Reply:
OK, as we discussed I will delete all the 32bit support libraries.